tutorials/workflow/csharp/child-workflows/ChildWorkflows/Program.cs

```diff
@@ -0,0 +1,25 @@
+using Dapr.Workflow;
+using ChildWorkflows;
+using ChildWorkflows.Activities;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddDaprWorkflow(options => {
+    options.RegisterWorkflow<ParentWorkflow>();
+    options.RegisterWorkflow<ChildWorkflow>();
+    options.RegisterActivity<Activity1>();
+    options.RegisterActivity<Activity2>();
+});
+var app = builder.Build();
+
+app.MapPost("/start", async (string[] input) => {
```

Comment:
No need for the await using var scope here as the inbound request will create a new scope itself. Rather have this:
```cs
app.MapPost("/start", async (string[] input, DaprWorkflowClient workflowClient) => {
    var instanceId = await workflowClient.ScheduleNewWorkflowAsync(
        name: nameof(ParentWorkflow),
        input: input);

    return Results.Accepted(instanceId);
});
```

Reply:
I had no idea, this makes it so much cleaner! 😍